tests/test-gguf.cpp

```diff
@@ -1051,13 +1049,13 @@ static bool same_tensor_data(const struct ggml_context * orig, const struct ggml
         }
 
         t_orig = ggml_get_next_tensor(orig, t_orig);
-        t_read = ggml_get_next_tensor(orig, t_read);
+        t_read = ggml_get_next_tensor(read, t_read);
```

Comment:
This is another bug fix, but it the `same_tensor_data` tests continues to fail. Will continue looking into this later. We can merge after it is fixed.